Monika-After-Story/MonikaModDev
# Update screen fixes

An attempt to fix the update bar and the updater's restart function refusing to work in windows.
## Testing:
- Verify the progress bar now works in all update stages (preparing, downloading and unpacking the update).
- Verify the `Quit` and `Restart` buttons appear when the update has finished, regardless if it's successful or not. The buttons should **not** appear during the update stages mentioned above, as long as in the `Checking for updates` and `Version x is available` screens.
- Verify the `Quit` and `Restart` buttons, at the end of the update, work as intended. (`Quit` should only close MAS while `Restart` should close the current MAS window and open a new one.)
- Verify no weird side effects by the constant call of `renpy.restart_interaction` every 0.1 secs (e.g. no lag, memory leaks, failed updates, pcs imploding etc.)
- Verify the progress bar and the restart function still works in Linux.
